client/components/common/ManageRequiredReportsDisclosure/index.jsx

```diff
@@ -0,0 +1,294 @@
+        let newCombinations = [...prevCombinations];
+
+        if (action === 'add') {
+          newCombinations.push({
+            at: ats.find(at => at.id === updatedReport.atId),
+            browser: browsers.find(
+              browser => browser.id === updatedReport.browserId
+            ),
+            phase: updatedReport.phase.replace('IS_', '')
+          });
+        }
+
+        newCombinations = filterAtBrowserCombinations(
+          newCombinations,
+          updatedReport
+        );
+        return sortAtBrowserCombinations(newCombinations);
+      });
+    },
+    [ats, browsers]
+  );
+
+  const handleRequiredReportAction = useCallback(
+    async (action, actionData, mutationFn, successMessage) => {
+      try {
+        await triggerLoad(async () => {
+          const { data } = await mutationFn({
+            variables: {
+              ...actionData,
+              phase: `IS_${actionData.phase}`
+            }
+          });
+
+          const updatedReport = data.requiredReport[`${action}RequiredReport`];
+
+          if (updatedReport) {
+            updateAtBrowserCombinations(updatedReport, action);
+          }
+
+          await triggerUpdate();
+        }, successMessage);
+      } catch (error) {
+        showErrorModal(
+          `Error ${
+            action.charAt(0).toUpperCase() + action.slice(1)
+          }ing Required Report`,
+          error.message
+        );
+      }
+    },
+    [triggerLoad, triggerUpdate, updateAtBrowserCombinations, showErrorModal]
+  );
+
+  const handleCreateRequiredReport = useCallback(
+    createData =>
+      handleRequiredReportAction(
+        'create',
+        createData,
+        createRequiredReport,
+        'Adding Phase requirement to the required reports table'
+      ),
+    [handleRequiredReportAction, createRequiredReport]
+  );
+
+  const handleDeleteReqReport = useCallback(
+    deletionData => {
+      handleRequiredReportAction(
+        'delete',
+        deletionData,
+        deleteRequiredReport,
+        'Deleting Phase requirement from the required reports table'
+      ),
+        handleCloseDeleteModal();
+    },
+    [handleRequiredReportAction, deleteRequiredReport, handleCloseDeleteModal]
+  );
+
+  const handleUpdateReport = useCallback(
+    updateData => {
+      handleRequiredReportAction(
+        'update',
+        updateData,
+        updateRequiredReport,
+        'Updating Phase requirement in the required reports table'
+      ),
+        handleCloseUpdateModal();
+    },
+    [handleRequiredReportAction, updateRequiredReport, handleCloseUpdateModal]
+  );
+
+  const handleCloseUpdateModal = () => {
+    setPendingUpdateReport(null);
+  };
+
+  const handleCloseDeleteModal = () => {
+    setPendingDeleteReport(null);
+  };
+
+  return (
+    <>
+      <DisclosureContainer data-testid="manage-required-reports-disclosure">
+        <span>Add required reports for a specific AT and Browser pair</span>
```

Comment:
```suggestion
        <span>Update which assistive technology and browser combinations require reports for the Candidate and Recommended phases</span>
```

May need some workshopping but this is for a bit more clarity